phpunit/tests/fonts/font-library/wpRestFontFamiliesController.php

```diff
@@ -331,7 +356,7 @@ public function test_create_item_with_default_preview( $settings ) {
 	public function data_create_item_with_default_preview() {
 		$default_settings = array(
 			'name'       => 'Open Sans',
-			'slug'       => 'open-sans',
+			'slug'       => 'create_item_with_default_preview',
```

Comment:
Just checking if the slug should be changed here.

Reply:
The slugs for test fixtures just need to be something unique, since that constraint is now added for font families. I'll make this less confusing by using `'open-sans-2'`, instead.